lib/experimental/Navigation/Dropdown/index.tsx

```diff
@@ -61,7 +61,7 @@ const DropdownItem = ({ item }: { item: DropdownItem }) => {
   )
 
   const itemClass = cn(
-    "flex items-start gap-1.5 w-full",
+    "cursor-pointer flex items-start gap-1.5 w-full",
```

Comment:
With the introduction of [this PR](https://github.com/factorialco/factorial-one/pull/914), I think this shouldn't be needed if these header actions are links, right?

Reply:
LOL we fixed it at the same time. I was debating myself whether to always make it `pointer` regardless of it being a link or not. Are we sure users know the difference? Anyways this will always be a link in this use case, so I imagine no harm done.